src/fontra/views/editor/edit-behavior.js

```diff
@@ -39,13 +40,20 @@ export class EditBehaviorFactory {
     this.components = unpackComponents(instance.components, relevantComponentIndices);
     this.anchors = unpackAnchors(instance.anchors, anchorSelection || []);
     this.guidelines = unpackGuidelines(instance.guidelines, guidelineSelection || []);
+    this.backgroundImage = backgroundImageSelection
+      ? instance.backgroundImage
+      : undefined;
     this.componentOriginIndices = componentOriginIndices || [];
     this.componentTCenterIndices = componentTCenterSelection || [];
     this.behaviors = {};
     this.enableScalingEdit = enableScalingEdit;
   }
 
-  getBehavior(behaviorName, fullComponentTransform = false) {
+  getBehavior(
+    behaviorName,
+    fullComponentTransform = false,
```

Comment:
Are you sure we need separate flags for fullComponentTransform and fullBackgroundImageTransform? They are the same thing (either both are true, or both are false), so maybe rename `fullComponentTransform` to `doFullTransform`, and you're covered for both cases.

Reply:
agree. done.